ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -61,7 +61,7 @@ effective_cache_size = {{ postgresql_effective_cache_size }}
 log_destination = 'csvlog'
 logging_collector = on
 log_directory = 'pg_log'
-log_filename = 'postgresql-%x-%r.log'
```

Comment:
before: could not open log file "pg_log/postgresql-11/07/17-07:37:32 PM.log": No such file or directory